### PR TITLE
feat(intersect) added intersect function …

### PR DESCRIPTION
**Pull request by dev-warner**
it creates a unique array of values that is contained in both arrays.

### CODE REVIEW

File: src/collection-proxy.js [hunk 1]
```diff
@@ -220,6 +220,16 @@ class CollectionProxy {
     )
   }
 
+  /**
+   * Creates an array of unique values that are included in both given array
+   * @param {Array} items
```
Comment (marcuspoehls) [inline src/collection-proxy.js:226]: Can you please add an empty line between the comment and the `@param`?

```suggestion
   *
   * @param {Array} items
```

File: src/collection.js [hunk 2]
```diff
@@ -231,6 +231,16 @@ class Collection {
     return !!item
   }
 
+  /**
+   * Creates an array of unique values that are included in both given array
+   * @param {Array} items
```
Comment (marcuspoehls) [inline src/collection.js:237]: Same here, can you please add an empty line before `@param`?

```suggestion
   *
   * @param {Array} items
```

**Review by marcuspoehls — CHANGES_REQUESTED** [review]
@dev-warner Thank you Joe for the PR! I'm requesting minor reformats. The implementation looks good! I like your test case to ensure that the initial collection still has the duplicate and the intersection does not. Nice!

**Comment by dev-warner** [comment]
Cheers let me know if there's anything else i can do :)

**Review by marcuspoehls — APPROVED** [review]

**Comment by marcuspoehls** [comment]
@dev-warner Thank you for the refinements Joe! Looks good, merged!

**Pull request merged**